tck/running_the_tck.asciidoc

```diff
@@ -109,3 +109,11 @@ Debugging is dependent on the TCK implementor. The implementor configures Arquil
 to deploy the test deployments and run the LRA client side implementation there. In general we can say that
 the runtime needs to define java debug properties `-Xdebug -Xrunjdwp:transport=dt_socket,address=8787,server=y,suspend=y`.
 Then debugger may be connected to port 8787 and to track the test execution.
+
+=== Additional notes
+
+* `TckTestBase#replayEndPhase(String)` method, used for `@Status` and `@Forget` call 
```

Comment:
It is also used for testing participants that compensate in the background.

Reply:
Thanks! I will add "for instance" in this sentence.